main.gms

```diff
@@ -531,7 +532,9 @@ $setGlobal cm_import_EU  off !! def off
 *** switch for ariadne import scenarios (needs cm_import_EU to be not off)
 $setGlobal cm_import_ariadne  off !! def off
 *** switch for ariadne energy security scenario
-$setGlobal cm_EnSecScen  off !! def off
+$setGlobal cm_EnSecScen  on !! def off
```

Comment:
Not sure, but don't we prefer the switches having the default values in main.gms?